File: Typescript/Typescript/evenOrOdd.ts
/*
Create a function that takes an integer as an argument and returns "Even" for even numbers or "Odd" for odd numbers.
*/

// My solution:
export function evenOrOdd(n:number):string {
    const type = n % 2 === 0 ? "Even" : "Odd";
    return type;
  }

// Tests:
console.log(evenOrOdd(5));
console.log(evenOrOdd(4));
console.log(evenOrOdd(0));
console.log(evenOrOdd(7));

File: Python/difficulty_eight_kyu/even_or_odd.py
'''
Create a function that takes an integer as an argument and returns "Even" for even numbers or "Odd" for odd numbers.
'''


# My solution:
def even_or_odd(number):
    return 'even' if number % 2 == 0 else 'odd'

# Tests:
print(even_or_odd(4))
print(even_or_odd(5))